tests/h/cli/commands/search_test.py

```diff
@@ -5,6 +5,11 @@
 
 from h.cli.commands import search
 
+pytestmark = [
+    pytest.mark.xdist_group("elasticsearch"),
+    pytest.mark.usefixtures("init_elasticsearch"),
+]
```

Comment:
As for the functests, putting this module-level `usefixtures` in each unittests module that actually needs the `init_elasticsearch` fixture.

The `xdist_group("elasticsearch")` tells `pytest-xdist` to send all tests in modules that have that mark to the same process. For Postgres we use isolated transactions to enable tests to be run in parallel even if they use Postgres but we don't have a similar solution for tests that use Elasticsearch so I'm using this `xdist_group` thing to prevent Elasticsearch-using tests from being run in parallel with each other.